docs/supported-languages.md

```diff
@@ -314,4 +314,16 @@ Semgrep Pro Engine offers support for the following languages:
 
 To install and run Semgrep Pro Engine, see [Semgrep Pro Engine overview](/deepsemgrep/deepsemgrep-introduction/).
 
+### Known Limitations
+
+<b>CommonJS</b>
+
+Currently Semgrep Pro does not handle all cases of CommmonJS imports, if you create a function and assign it to an export later Semgrep Pro does not capture this today, read [this page](https://semgrep.dev/docs/deepsemgrep/deepsemgrep-examples/#es6-and-commonjs) for more information.
+
+<b>Regressions in Semgrep Pro</b>
+
+For interfile analysis, Semgrep Pro Engine resolves names differently than Semgrep and this may cause rules with `interfile: true` to produce different results than Semgrep OSS Engine. Some instances could be regarded as regressions, if you encounter them please file a bug report.
```

Comment:
@LewisArdern where do we recommend users submit Semgrep Pro Engine bug reports?

Reply:
@LewisArdern @IagoAbal to specify: In which platform do we recommend users to submit tickets? Should they submit issues to regular Sermgrep OSS repository even for Semgrep Pro Engine issues?